docs/recipes/babelrc.md

```diff
@@ -30,6 +30,21 @@ You can override the default Babel configuration AVA uses for test transpilation
 }
 ```
 
+## Provide ES2015 Polyfills to your tests
+
+You might run into issues when relying on ES2015-features your current environment do not support. Since Ava isn't providing any polyfills per default you might want to enable them by adding `babel-polyfill` manually to the `require` section:
```

Comment:
Thank you very much @novemberborn - i somehow assumed it could be wanted to write not too much. 🤔 I guess that assumption was just a bit stupid. 😄

- [ ] Elaborate the description